test/FIFOMarket.test.ts

```diff
@@ -212,13 +229,17 @@ describe('FIFOMarket', () => {
       const { supplier, buyer, noriWallet } = hre.namedAccounts;
 
       const removalBalances = [];
-      const vintages = [];
-      const tokenIds = [];
+      let tokenIds = [];
       for (let i = 0; i <= 20; i++) {
         removalBalances.push(hre.ethers.utils.parseUnits('50'));
-        vintages.push(2018);
-        tokenIds.push(i);
+        tokenIds.push(
```

Comment:
is it worth turning this into a util? A previous test in this file has similar setup logic but doesn't set up using the same pattern (`should purchase removals and mint a certificate for a small purchase spanning several removal`)